### PR TITLE
CIT-41 #time 3h create comment model, create API to get comments of a…

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
… particular blog using APIView class

### CODE REVIEW

**Comment by codal-uchoudhary** [comment]
11_march :
- Implement password validation function
- create comments model
- create an API to get comment of a specific blog using APIView class

Note : 
-  I will update my API from APIView class to ModelViewSet class  and add create method for adding comments in blog in next PR. i implement APIView class here to understand the concept

**Comment by codal-uchoudhary** [comment]
image

**Comment by codal-uchoudhary** [comment]
Get API for comments is working for now. as we look in above Screen-Shot

File: Projects/My_website/myWebsite/account/utils.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import re
+
+
+# function for validation of strong password
+def StrongPasswordValidator(password):
```
Comment (abhi-codal) [inline Projects/My_website/myWebsite/account/utils.py]: @codal-uchoudhary please use snack cases. this func name should be `strong_password_validator`

File: Projects/My_website/myWebsite/account/serializers.py [hunk 2]
```diff
@@ -24,6 +25,11 @@ def validate(self, data):
             if User.objects.filter(email=data["email"]).exists():
                 raise serializers.ValidationError("email is exist")
 
+        if data["password"]:
+            status, message = StrongPasswordValidator(data["password"])
```
Comment (abhi-codal) [inline Projects/My_website/myWebsite/account/serializers.py]: @codal-uchoudhary can you please use `is_valid` instead of status, as it makes more sense?

**Review by abhi-codal — APPROVED** [review]

**Pull request merged**